recipes/blake2/all/conanfile.py

```diff
@@ -0,0 +1,70 @@
+import os
+import stat
+import shutil
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+
+class blake2Conan(ConanFile):
+    name = "blake2"
+    version = "997fa5b" #BLAKE2 doesn't have versions so we use the commit hash instead
+    license = "	CC0-1.0, OpenSSL, APSL 2.0"
```

Comment:
```suggestion
    license = ["CC0-1.0", "OpenSSL", "APSL-2.0"]
```

Is only about Creative Commons: https://github.com/BLAKE2/BLAKE2/blob/master/COPYING

Reply:
[here](https://github.com/BLAKE2/BLAKE2/blob/master/README.md) says it is triple licenced and [here](https://blake2.net/) only talks about Creative Commons. I'm really not sure which one to choose.

Reply:
Keep it as is, but use a list instead.